docs/API.md

```diff
@@ -451,77 +499,146 @@ New version:
     int srt_setsockflag(SRTSOCKET socket, SRT_SOCKOPT optName, const void* optval, int optlen);
 
 (In the legacy version, there's an additional unused `level` parameter. It was there
-in the original UDT API just to mimic the system `setsockopt` function).
-
-Some options require a value of type bool and some others of type int, which is
-not the same -- they differ in size, and mistaking them may end up with a crash.
-This must be kept in mind especially in any C wrapper. For convenience, the
-setting option function may accept both `int` and `bool` types, but this is
-not so in the case of getting an option value.
-
-Almost all options from the UDT library are derived (there are a few deleted, including
-some deprecated already in UDT), many new SRT options have been added.
-All options are available exclusively with the `SRTO_` prefix. Old names are provided as
-alias names in the `udt.h` legacy/C++ API file. Note the translation rules:
+in the original UDT API just to mimic the system `setsockopt` function, but it's ignored).
+
+Some options require a value of type bool and some others of an integer type,
+which is not the same -- they differ in size, and mistaking them may end up
+with a crash. This must be kept in mind especially in any C wrapper. For
+convenience, the setting option function may accept both `int32_t` and `bool`
+types, but this is not so in the case of getting an option value.
+
+**UDT project legacy note**: Almost all options from the UDT library are
+derived (there are a few deleted, including some deprecated already in UDT),
+many new SRT options have been added. All options are available exclusively
+with the `SRTO_` prefix. Old names are provided as alias names in the `udt.h`
+legacy/C++ API file. Note the translation rules:
 * `UDT_` prefix from UDT options was changed to the prefix `SRTO_`
 * `UDP_` prefix from UDT options was changed to the prefix `SRTO_UDP_`
 * `SRT_` prefix in older SRT versions was changed to `SRTO_`
 
-The Binding column should define for these options one of the following
-statements concerning setting a value:
+The below table shows the characteristics of the options. Legend:
+
+1. **Since**
+
+Defines the SRT version when this option was first introduced. If this field
+is empty, it's an option derived from UDT. Version 0.0.0 is the oldest version
```

Comment:
Version 0.0.0? Is this a placeholder?

Reply:
It's simply stating that the actual version when it was introduced is unknown, but it's stated that it's there since always.

Reply:
`is empty, it's an option derived from UDT. "Version 0.0.0" refers to the oldest version
`